app/Services/TokenManager.php

```
@@ -0,0 +1,172 @@
+        Log::info('Refreshing GoCardless access token', ['user_id' => $this->user->id]);
+
+        $response = Http::post("{$this->baseUrl}/token/refresh/", [
+            'refresh' => $this->user->gocardless_refresh_token,
+        ]);
+
+        if (!$response->successful()) {
+            throw new \Exception('Failed to refresh access token: ' . $response->body());
+        }
+
+        $data = $response->json();
+        $this->updateTokens($data);
```

PHPStan (2.1.15)

Parameter #1 $tokenData of method App\Services\TokenManager::updateTokens() expects array, mixed given.


(argument.type)